core/src/Revolution/Processors/Element/Create.php

```diff
@@ -29,6 +29,61 @@ abstract class Create extends CreateProcessor
     /** @var modElement $object */
     public $object;
 
+    protected $hasStaticFile = false;
+
+    public function initialize()
+    {
+        // Intitializing parent first, as we need the Element object created before moving forward
+        if (parent::initialize()) {
+            $className = array_pop(explode('\\', $this->classKey));
```

Comment:
Only variables can be passed by reference. You need to introduce a variable here and pass that to `array_pop`.